--- libxml2-2.13.0/xmlIO.c.orig	2024-06-14 12:32:12.257784372 +0200
+++ libxml2-2.13.0/xmlIO.c	2024-06-14 19:23:06.355299914 +0200
@@ -547,6 +547,29 @@
     fd = open(filename, flags, 0777);
 #endif /* WIN32 */
 
+#ifdef LIBXML_ZLIB_ENABLED
+    if (fd < 0 && errno == ENOENT) {
+        /* If we cannot open `foo.xml', we try `foo.xml.gz'. However we i
+         * append .gz only if filename doesn't alreay end with .gz. */
+        if (strlen(filename) < 3 || strcmp(filename + strlen(filename) - 3, ".gz") != 0) {
+            char *gz_filename = xmlMalloc(strlen(filename) + 4);
+            int gz_fd;
+
+            if (gz_filename != NULL) {
+                strcpy(gz_filename, filename);
+                strcat(gz_filename, ".gz");
+            }
+
+            gz_fd = open(gz_filename, flags, 0777);
+
+            if (gz_fd >= 0)
+                fd = gz_fd;
+
+            xmlFree(gz_filename);
+        }
+    }
+
+#endif /* LIBXML_ZLIB_ENABLED */
     if (fd < 0) {
         /*
          * Windows and possibly other platforms return EINVAL
