--- libxml2-2.13.1/dict.c.orig	2024-06-12 13:17:43.000000000 +0200
+++ libxml2-2.13.1/dict.c	2024-06-19 21:40:34.925985581 +0200
@@ -927,7 +927,6 @@ xmlDictQLookup(xmlDictPtr dict, const xm
 #ifdef _WIN32
   #define WIN32_LEAN_AND_MEAN
   #include <windows.h>
-  #include <bcrypt.h>
 #elif defined(HAVE_GETENTROPY)
   #ifdef HAVE_UNISTD_H
     #include <unistd.h>
@@ -954,7 +953,7 @@ xmlInitRandom(void) {
     xmlInitMutex(&xmlRngMutex);
 
     {
-#ifdef _WIN32
+#ifdef TOONEW_WIN32
         NTSTATUS status;
 
         status = BCryptGenRandom(NULL, (unsigned char *) globalRngState,
--- libxml2-2.13.1/configure.ac.orig	2024-06-19 12:13:03.000000000 +0200
+++ libxml2-2.13.1/configure.ac	2024-06-19 21:49:19.486477127 +0200
@@ -1083,7 +1083,7 @@ dnl
 dnl Crypto libraries
 dnl
 case "$host" in
-    *-*-mingw*)
+    *-*-mingw*64*)
         CRYPTO_LIBS="-lbcrypt"
         ;;
 esac
